src/renderers/native/ReactNative.js

```diff
@@ -21,13 +21,15 @@ var ReactUpdates = require('ReactUpdates');
 
 var findNodeHandle = require('findNodeHandle');
 
+import type { ReactElement } from 'ReactElementType';
+
 ReactNativeDefaultInjection.inject();
 
 var render = function(
-  element: ReactElement<any>,
+  element: ReactElement | string | number | null | false,
```

Comment:
Is this the correct type here in render? Should we make this verbose thing a named type on its own as we start to use it in a fair amount of places? If yes, what should we call it?